docs/reference/sso.md

```diff
@@ -0,0 +1,522 @@
+  # Important: make sure to include the rest of your lakeFS Configuration here!
+
+  auth:
+    remote_authenticator:
+      enabled: true
+      endpoint: https://lakefs.company.com/api/v1/ldap/login
+      default_user_group: "Developers"
+    ui_config:
+      logout_url: /logout
+      login_cookie_names:
+        - internal_auth_session
+      rbac: external
+```
+
+Fluffy Configuration:
+```yaml
+fluffyConfig: &fluffyConfig |
+  logging:
+    format: "json"
+    level: "INFO"
+    audit_log_level: "INFO"
+    output: "="
+  installation:
+    fixed_id: fluffy-authenticator
+  auth:
+    post_login_redirect_url: /
+    ldap: 
+      server_endpoint: 'ldaps://ldap.company.com:636'
+      bind_dn: uid=<bind-user-name>,ou=Users,o=<org-id>,dc=<company>,dc=com
+      bind_password: '<ldap pwd>'
+      username_attribute: uid
+      user_base_dn: ou=Users,o=<org-id>,dc=<company>,dc=com
+      user_filter: (objectClass=inetOrgPerson)
+      connection_timeout_seconds: 15
+      request_timeout_seconds: 7
+```
+
+### Helm
+
+In order to use lakeFS Enterprise and Fluffy, see [lakeFS Helm chart configuration](https://github.com/treeverse/charts/tree/master/charts/lakefs).
+
+Notes:
+* `secrets.authEncryptSecretKey` is shared between fluffy & lakeFS and must be equal 
+* lakeFS `image.tag` must be >= 0.95.0
+* Change the `fluffy.ingress.host` from `lakefs.company.com` to a real host (usually same as lakeFS), also update additional references in the file (note: URL path after host if provided should stay unchanged).
+* Update the `fluffy.ingress` configuration with other optional fields if used
+* Fluffy docker image: replace the `fluffy.image.secretToken` with real token to dockerhub for the fluffy docker image.
+
+```yaml
+{% raw %}###############################################
+###############################################
+############ COMMON CONFIGURATION #############
+###############################################
+###############################################
+
+fluffy:
+  serviceAccountName: default
+  name: &fluffyName lakefs-fluffy
+  logLevel: INFO
+  replicaCount: &fluffyReplicaCount 1
+  resources: &resources
+    limits:
+      memory: 256Mi
+    requests:
+      memory: 128Mi
+  ingress:
+    enabled: &enableIngress true 
+    host: &ingressHost lakefs.company.com
+    ingressClassName: &ingressClassName ''
+    annotations: &ingressAnnotations {}
+  image: 
+    repository: 'treeverse/fluffy'
+    tag: '0.1.1'
+    secretName: 'fluffy-dockerhub'
+    secretToken: &fluffyImageSecretToken '<docker token>'
+  saml_rsa_public_cert: |
+    -----BEGIN CERTIFICATE-----
+    -----END CERTIFICATE-----
+  saml_rsa_private_key: |
+    -----BEGIN PRIVATE KEY-----
+    -----END PRIVATE KEY-----
+
+###############################################
+###############################################
+###### ADVANCED DON'T EDIT VALUES BELOW #######
+###############################################
+###############################################
```

Comment:
why are we putting this in the docs if all the user has to do is copy and paste it and not change it? wouldn't it be better to ship a config file and then tell them where to find it and what to change?

Reply:
they can edit the part below if they like, it's advanced configuration, we are planning on releasing a dedicated chart for the SSO service that will reduce the need of these configuration snippets